compose/cli/log_printer.py

```diff
@@ -175,9 +175,8 @@ def build_log_generator(container, log_args):
     # if the container doesn't have a log_stream we need to attach to container
     # before log printer starts running
     if container.log_stream is None:
-        stream = container.logs(stdout=True, stderr=True, stream=True, **log_args)
```

Comment:
This patch ignores `log_args`, so docker-compose logs is perhaps broken. I'll fix this.